client/src/main/java/org/apache/rocketmq/client/consumer/DefaultMQPullConsumer.java

```diff
@@ -250,6 +265,7 @@ public void setRegisterTopics(Set<String> registerTopics) {
     @Override
     public void sendMessageBack(MessageExt msg, int delayLevel)
         throws RemotingException, MQBrokerException, InterruptedException, MQClientException {
+        msg.setTopic(withNamespace(msg.getTopic()));
```

Comment:
Check msg is null and throw MQClientException if necessary.

Reply:
yes, this issue has been resolved,  @ShannonDing added the check logic before send message